### ISSUE TITLE
[Feature request] Allow unpacked HDD games to be installed like disc games

### ISSUE THREAD

**Issue by Rex000** [RPCS3/rpcs3#13391]
### Quick summary

Disc games can be installed easily in folder form using the "Add game" feature. Same should be allowed for HDD games.

### Details

In my experience, having HDD games and all their respective updates unpacked and merged into one folder gives several benefits, some of which are listed below:

- Saves the hassle of installing the games then installing their updates all over again upon a clean install of RPCS3.
- Greatly saves space since one doesn't need to keep .pkg backups for games and updates.
- Allows for proper custom naming and organizing of game folders instead of just having Title ID as their folder names.
- One can even keep all their games in one place if they like, or keep them separately, regardless of whether they are HDD games or disc games.

I have tried this myself and the games work fine, but the issue is that trial games stay as trial. Copying their edat files to RPCS3's edata folder brought up a "Booting failed: No bootable content was found" error.

In short, this feature would bring about a great level of flexibility and ease of use for the end user with several benefits.

**Comment by Rex000** [comment]
Duplicate of what issue?

Also sorry for using the wrong word. I meant "added". I hope it's the correct word.

**Comment by Megamouse** [comment]
You literally used this feature to add your games in your other issue. i don't know what else you want.

**Comment by Rex000** [comment]
I'll delete that issue if you want me to. As you can in that issue I was told by [MSuih](https://github.com/MSuih) that it's not an officially supported feature hence why I created a request for it.
